e2e/cypress/integration/enterprise/cloud/Billing/company_Information_spec.js

```diff
@@ -0,0 +1,131 @@
+
+    it('MM-37051 - Save Info should persist the data and same persisted data should be displayed in Company Information screen', () => {
+        const compnayName = getRandomLetter(30);
+        cy.visit('/admin_console/billing/company_info');
+
+        // # click on edit Company Information button
+        cy.get('.CompanyInfoDisplay__companyInfo-editButton').click();
+
+        // # enter company information functionality
+        cy.get('#input_companyName').clear().type(compnayName);
+        cy.get('#input_numEmployees').clear().type('10');
+        cy.get('.css-1hwfws3').click();
+        cy.get(".DropDown__input > input[type='text']").type('India{enter}');
+        cy.get('#input_address').clear().type('test address');
+        cy.get('#input_address2').clear().type('test2');
+        cy.get('#input_city').clear().type('testcity');
+        cy.get('#input_state').clear().type('test');
+        cy.get('#input_postalCode').clear().type('44455');
+
+        // # click on Save Info button
+        cy.get('#saveSetting').should('be.enabled').click();
+
+        // * check for persisted company name
+        cy.get('.CompanyInfoDisplay__companyInfo-name').should('have.text', compnayName);
+
+        // * check for employee number
+        cy.get('.CompanyInfoDisplay__companyInfo-numEmployees > span').should('include.text', '10');
+
+        // * check for country
+        cy.get('.CompanyInfoDisplay__companyInfo-address > div').eq(3).should('have.text', 'British Indian Ocean Territory');
+
+        // * check for city,state and postol code
+        cy.get('.CompanyInfoDisplay__companyInfo-address > div').eq(2).should('have.text', 'testcity, test, 44455');
+
+        // * check for address 2
+        cy.get('.CompanyInfoDisplay__companyInfo-address > div').eq(1).should('have.text', 'test2');
+
+        // * check for address 1
+        cy.get('.CompanyInfoDisplay__companyInfo-address > div').eq(0).should('have.text', 'test address');
+    });
+
+    it('MM-37051 - Cancelling of editing of company information details', () => {
+        cy.visit('/admin_console/billing/company_info');
+
+        // # click Add edit Information button
+        cy.get('.CompanyInfoDisplay__companyInfo-editButton').click();
+
+        // # click back button of Edit Company Information
+        cy.contains('span', 'Edit Company Information').prev().click();
+
+        // * check for back functionality using back button of edit company information screen
+        cy.get('.CompanyInfoDisplay__companyInfo-editButton').should('be.visible');
+
+        // # click Add Company Information button
+        cy.get('.CompanyInfoDisplay__companyInfo-editButton').click();
+
+        // # click cancel button
+        cy.contains('span', 'Cancel').parent().click();
+
+        // * check for visibility of Add Company Information button
+        cy.get('.CompanyInfoDisplay__companyInfo-editButton').should('be.visible');
+    });
+});
+
+const gotoCompanyInformationScreen = () => {
```

Comment:
this might be a ui command so other test can use it if needed

Reply:
gotoCompanyInformationScreen function is specific to Company Information screen So it is present locally